donut.js

```diff
@@ -7,42 +7,59 @@
      this.hourlyDonuts = []
    }
 
```

Comment:
I won't say much because I've already given you lots of feedback on the topic, but be sure to double check your whitespace and indentations before submitting. You should be using 2-Spaced Indents. Do a quick google search for how to set your user prefs in sublime so that this happens automatically. It'll help a lot!